pkg/services/searchV2/bluge.go

```diff
@@ -595,6 +596,7 @@ func doSearchQuery(
 		}
 
 		if len(tags) > 0 {
+			sort.Strings(tags)
```

Comment:
Not sure it's the best place to do the sort, but I couldn't find a better solution.